doc/modules/mixture.rst

```diff
@@ -189,11 +210,11 @@ Pros
    variational solutions have less pathological special cases than
    expectation-maximization solutions.
 
-:Automatic selection: when `dirichlet_concentration_prior` is small enough and
-   `n_components` is larger than what is found necessary by the model, the
+:Automatic selection: when ``weight_concentration_prior`` is small enough and
+   ``n_components`` is larger than what is found necessary by the model, the
    Variational Bayesian mixture model has a natural tendency to set some mixture
-   weights values close to zero. This makes it possible to let the model choose a
-   suitable number of effective components automatically.
+   weights values close to zero. This makes it possible to let the model choose
+   a suitable number of effective components automatically.
```

Comment:
Please remove / merge the pros and the cons section with the one for the Dirichlet process variant.